components/ai_chat/resources/page/state/data-context-provider.tsx

```diff
@@ -191,9 +191,11 @@ function DataContextProvider (props: DataContextProviderProps) {
     setCanShowPremiumPrompt(false)
   }
 
+  // TODO(petemill): rename to switchToNonPremiumModel as there are no longer
+  // a different in limitations between basic and freemium models.
```

Comment:
why aren't you renaming the function in this PR?

Reply:
Because this is an issue that needs uplifting